recipe/meta.yaml

```
@@ -50,7 +50,7 @@ outputs:
         - python >=3.10
         - openff-nagl-models >=0.1.0
         - pyarrow
-        - dgl >=1.0
+        - dgl >=1.0,<=2.1
```

```suggestion
        - dgl >=1.0,<=2.1
```

Added an upper pin just to be safe. It's much easier to put up a new build with a looser pin than the other way around.